pkg/model/resource/options.go

```diff
@@ -165,9 +214,15 @@ func (opts *Options) NewResource(c *config.Config, doResource bool) *Resource {
 
 	replacer := res.Replacer()
 
-	pkg := replacer.Replace(path.Join(c.Repo, "api", "%[version]"))
+	// selecting the api path based on multigroup or singlegroup project
+	apiPath := "api"
 	if c.MultiGroup {
-		pkg = replacer.Replace(path.Join(c.Repo, "apis", "%[group]", "%[version]"))
+		apiPath = "apis"
+	}
+
+	pkg := replacer.Replace(path.Join(c.Repo, apiPath, "%[version]"))
+	if c.MultiGroup && opts.Group != "" {
+		pkg = replacer.Replace(path.Join(c.Repo, apiPath, "%[group]", "%[version]"))
 	}
```

Comment:
Nit we can simplify the change here by just adding to the original code the additional condition. See; 

```
	pkg := replacer.Replace(path.Join(c.Repo, "api", "%[version]"))
	if c.MultiGroup && opts.Group != ""  {
		pkg = replacer.Replace(path.Join(c.Repo, "apis", "%[group]", "%[version]"))
	}
	domain := c.Domain
```

<img width="1347" alt="Screen Shot 2020-09-16 at 10 50 28" src="https://user-images.githubusercontent.com/7708031/93321743-74227c80-f80a-11ea-86d7-87230c0e0adc.png">

Reply:
There are two path right now, for single group "api" and for multigroup project "apis". If we go by above logic, the multigroup project with empty group will take the path as "api" instead of "apis". That is why I choose to have apiPath as "api" and "apis" based on the single or multi group first.

Reply:
It is perfect tks you can close this one.